cypress/integration/signup-page-spec.js

```diff
@@ -0,0 +1,78 @@
+describe('Test signup page', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/signup/');
+  });
+
+  it('Test text', () => {
+    cy.contains('SIGN UP');
+    cy.contains('NAME');
+    cy.contains('EMAIL');
+    cy.contains('PASSWORD');
+    cy.contains('CONFIRM PASSWORD');
+    cy.contains('GENDER');
+    cy.get('input[name="password1"]').should(
+      'have.attr',
+      'placeholder',
+      'MIN. 8 CHARACTERS LONG'
+    );
+  });
+
+  it('Test signup inputs', () => {
+    cy.get('input[name="name"]')
+      .type('Fake Name')
+      .should('have.value', 'Fake Name');
+    cy.get('input[name="email"]')
+      .type('fake@email.com')
+      .should('have.value', 'fake@email.com');
+    cy.get('input[name="password1"]')
+      .type('fakepassword')
+      .should('have.value', 'fakepassword');
+    cy.get('input[name="password2"]')
+      .type('fakepassword')
+      .should('have.value', 'fakepassword');
+    cy.get('#react-select-2-input').type('MALE{enter}', { force: true });
+    cy.get('#react-select-2-input').type('FEMALE{enter}', { force: true });
+    cy.get('#react-select-2-input').type('NOT SPECIFIED{enter}', {
+      force: true,
+    });
```

Comment:
what is the assertion here?

Reply:
That the selection in the react select is correct.
Do you know other way of test this?